src/github/dto/github.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+class CommunityStats {
+  [key: string]: number;
```

Comment:
This class definition should be adjusted depending on the outcome of the question in #63